### DISCUSSION TITLE
日志没有限制导致chrome崩溃

### DISCUSSION THREAD

**Discussion by Calcifer97** [1Panel-dev/1Panel#5112] [Q&A]
### 联系方式

zhanglor@163.com

### 1Panel 版本

v1.10.7-lts

### 问题描述

日志太多了。导致chrome崩溃
![image](https://github.com/1Panel-dev/1Panel/assets/37408196/7806f86c-1bc8-4b06-818f-06dcf1d6e2fc)
![image](https://github.com/1Panel-dev/1Panel/assets/37408196/a869c009-aa3e-4c5c-9b5c-774efe11ed31)

### 重现步骤

必现的，只要日志多就可以

### 期待的正确结果

如果日志太多，就只显示最近的

**Comment by sch-lda** [comment]
网站日志很多似乎没有办法，肯定是每个请求都要记录一次。建议增加日志分割设置，允许按天/周/月等时间段滚动
现在是用corn+shell脚本凑合着
```
#!/bin/bash

logdir="/opt/1panel/apps/openresty/openresty/www/sites/*.*.*/log"

fmt_date=$(date +"%Y-%m-%d")

cp ${logdir}/access.log ${logdir}/access_old_${fmt_date}.log
echo "" > ${logdir}/access.log
```
